### PR TITLE
Ms1 setup project

### PR DESCRIPTION
**Pull request by Bishoy-Samwel**
- [x] Create an npm project with webpack. 
- [x] Write plain HTML markup with minimum styling (just to make the layout)

### CODE REVIEW

File: .hintrc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  "formatters": ["stylish"],
+  "hints": [
+    "button-type",
+    "disown-opener",
+    "html-checker",
+    "meta-charset-utf-8",
+    "meta-viewport",
+    "no-inline-styles:error"
+  ]
+}
```
Comment (Dipeshtwis) [inline .hintrc]: - [ ] I would recommend removing this file as you are no longer using `HTML` linters.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @Bishoy-Samwel, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Used webpack to bundle JavaScript.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+<body>
+  <header>
+    <h1>Leaderboard</h1>
+  </header>
+  <main class="d-flex">
+    <div class="flex-1" id="table-container">
+      <div class="d-flex" id = "table-header">
+        <h2>Recent scores</h2>
+        <button id="refresh-btn">Refresh!</button>
+      </div>
```
Comment (henatan99) [inline src/index.html:19]: - [x] Please add a list of recent scores as shown in the wireframe example. Everything else is great :clap: . Kindly make this change and you are good togo.

image

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @Bishoy-Samwel , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mimipeshy — APPROVED** [review]
## Hi @Bishoy-Samwel 👋🏾 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**